<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SpaceWire-PCIe接口卡控制软件</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_openDev">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>打开设备</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>191</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>状态信息</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>561</width>
      <height>271</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>端口1</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>555</width>
        <height>246</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>端口2</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>555</width>
        <height>246</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>端口3</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>555</width>
        <height>246</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>端口4</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>555</width>
        <height>246</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>端口5</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>555</width>
        <height>246</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>端口6</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>555</width>
        <height>246</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>端口7</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>555</width>
        <height>246</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>端口8</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>555</width>
        <height>246</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_linkEnable">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>链路使能</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_linkreset">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>链路复位</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_foundnet">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>网络发现</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>50</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>200Mbps</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100Mbps</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50Mbps</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>传输速度</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="pixmap">
     <pixmap resource="untitled.qrc">:/red.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>295</x>
      <y>90</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="pixmap">
     <pixmap resource="untitled.qrc">:/red.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="pixmap">
     <pixmap resource="untitled.qrc">:/red.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>90</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="pixmap">
     <pixmap resource="untitled.qrc">:/red.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>90</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="pixmap">
     <pixmap resource="untitled.qrc">:/red.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>515</x>
      <y>90</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="pixmap">
     <pixmap resource="untitled.qrc">:/red.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>90</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="pixmap">
     <pixmap resource="untitled.qrc">:/red.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>625</x>
      <y>90</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="pixmap">
     <pixmap resource="untitled.qrc">:/red.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Modern No. 20</family>
      <pointsize>12</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>HIT-ATCI</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_readData">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>读取数据</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设备</string>
    </property>
    <addaction name="actionThree"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="actionPCIe"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionHelp"/>
    <addaction name="actionVersion"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionThree">
   <property name="text">
    <string>三轴加速度计</string>
   </property>
  </action>
  <action name="actionPCIe">
   <property name="text">
    <string>PCIe</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>使用帮助</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="text">
    <string>版本信息</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="untitled.qrc"/>
 </resources>
 <connections/>
</ui>
